lib/Data/Chronicle/Reader.pm

```diff
@@ -90,10 +66,11 @@ Given a category, name and timestamp returns version of data under "category::na
 
 =cut
 
+=head1 Members
+
 =head2 cache_reader
 
-cache_reader can be an object which has `get` method used to fetch data.
-or it can be a plain hash-ref.
+cache_reader should be an object of RedisDB.
```

Comment:
More accurately wording: "should be an instance of L<RedisDB>" - also using `L<...>` to link to the class docs.